### ISSUE TITLE
Customize frequently used columns

### ISSUE THREAD

**Issue by osdiab** [hasura/graphql-engine#9724]
### Is your proposal related to a problem?

On our project we use a different id algorithm than a sequential id, UUIDs etc. So I want my developers to reach for our standard, not the defaults which are inconsistent with our design choices.

### Describe the solution you'd like

Add some configuration, perhaps to metadata, where we can enumerate the frequently used columns list instead of depending on those hardcoded into the console.

**Comment by manasag** [comment]
Hi @osdiab, thanks for raising this issue. Can you provide some more details to which Console behavior you are referring here? Is it the "frequently used column" button which appears in "Add a new column" section of "Modify" a table?

**Comment by osdiab** [comment]
Yes that, as well as when creating a new table there is a frequently used column button as well for adding in e.g. generated IDs, created_at/updated_at timestamps, etc. I'd love to be able to remove all the stuff we don't use (e.g. integer IDs) and include things we do want to use (e.g. ULID ID column)

**Comment by ryparker** [comment]
Made a discussion specific to using custom id gen functions
https://github.com/hasura/graphql-engine/discussions/10488
